eZ/Publish/Core/MVC/Symfony/EventListener/Tests/SiteAccessMatchListenerTest.php

```diff
@@ -63,19 +61,15 @@ public function testGetSubscribedEvents()
 
     public function testOnKernelRequestUserHashNoOriginalRequest()
     {
+        $this->markTestSkipped('Test skipped until HttpCache is compatible with eZ Platform 3');
+
         $request = new Request();
         $event = new GetResponseEvent(
             $this->createMock(HttpKernelInterface::class),
             $request,
             HttpKernelInterface::MASTER_REQUEST
         );
 
-        $this->userHashMatcher
```

Comment:
I understand that `userContextRequestMatcher` in current version is not compatible with sf4, and it will be back with fosCache 2.X.
If yes and we have marked this test as skipped do we have to also remove this code?

If I am wrong about above, just ignore ;)

Reply:
This test is still valid without userHashMatcher this is why I'd skip this test. This change is a followup to the removed dependency on UserHashMatcher in `SiteAccessMatchListener`. Once you add it back you will have to align tests as well.